### ISSUE TITLE
[Admin events] Checking for set inclusion in expressions

### ISSUE THREAD

**Issue by mwakaba2** [arbitraryexecution/forta-agent-templates#23]
Hi, I'd like to use the admin events template to create a detection bot that only emits an alert for transfers/approvals associated with a specific set of addresses. Can I check for a set inclusion with the expression? Does the expression handle Ors?

Something like this:

```javascript
"Transfer": {
          "expression": "to == address_A || to == address_B || to == address_C",
}
```

Or

```javascript
"Transfer": {
          "expression": "to in address_A,address_B,address_C"
}
```

**Comment by echoAE** [comment]
No, not currently. The expression parsing only supports the following format:
`variableName operator operand`

For example, these address expressions would be supported:
`to === 0xc2a1ba2B511A9533972d4AdC02005d4192918753`
`from !== 0xc2a1ba2B511A9533972d4AdC02005d4192918753`

The chart at the bottom of this page shows the supported expressions and types:
https://github.com/arbitraryexecution/forta-agent-templates/blob/main/admin-events/SETUP.md

Adding support for a logical OR sounds reasonable without overly complicating the expression parsing. Having said that, we don't currently have a ticket for updating the expression parsing.

We intentionally limited the complexity of the expression parsing for the creation of the bot templates so that we could create more templates in the time provided.  We understood that there would be a desire to expand in the future to support more complex expressions.

We anticipated that, in the limit, people would request support for expressions that involve calling functions to evaluate the expression, which we are very leery of. Our goal with these templates was, to the extent possible, separate the logic and data concerns.  However, it seems that there is an ever-present desire to then inject logic through the data file.

We have looked at the npm package offered by Lazaro at Nethermind here as a possible drop-in replacement for our current simple expression parser:
https://www.npmjs.com/package/eth-expression-parser
https://github.com/e1Ru1o/eth-expression-parser
